lib/cadet/assessments/assessments.ex

```diff
@@ -421,13 +494,13 @@ defmodule Cadet.Assessments do
   end
 
   @doc """
-  Function returning submissions under a grader. 
```

Comment:
You should check what's causing this. Is it CRLF vs LF? You shld revert these useless changes cos it's noise on the diff haha